src/components/Popup/Popup.tsx

```diff
@@ -129,6 +129,7 @@ export default class Popup extends AutoControlledComponent<Extendable<PopupProps
       this.trySetOpen(!this.state.open, e, true)
     },
     closeAndFocusTrigger: e => this.closeAndFocusTrigger(e),
+    stopPropagation: e => e.stopPropagation(),
```

Comment:
I don't think that this concept is part of accessibility behavior, isn't it - otherwise we will need to decide the same way for `toggle` as well. We had already similar discussion with Juraj and had decided to leave event propagation aspects to the client.

More than that, looking on the implementation here (https://github.com/stardust-ui/react/pull/515/files#diff-1942cc0b79094f8768e5b6847613a941R28) I have concerns about the fact that multiple actions will be called for the same event _without any prescribed order suggested by the API_ - I would rather avoid to introduce this from happening.

I would rather suggest to make explicit `e.stopPropagation()` call in the `closeAndFocusTrigger` processing logic.

Reply:
I was thinking to inline this into the already existing handler but thought having it in behavior can bring more flexibility for us.
Ok, then will move it to ```closeAndFocusTrigger``` action handler for now.